File: src/AC/1460/1460.cpp
#include <algorithm>
#include <vector>

class Solution {
public:
    bool canBeEqual(std::vector<int>& target, std::vector<int>& arr) {
        std::vector<int> sorted_target = target;
        std::vector<int> sorted_arr = arr;
        std::sort(sorted_target.begin(), sorted_target.end());
        std::sort(sorted_arr.begin(), sorted_arr.end());
        return sorted_target == sorted_arr;
    }
};

File: src/AC/1460/1460.rs
impl Solution {
    pub fn can_be_equal(target: Vec<i32>, arr: Vec<i32>) -> bool {
        return target.len() == arr.len() && {
            let mut target = target;
            let mut arr = arr;
            target.sort();
            arr.sort();
            target == arr
        };
    }
}
